cms/templates/catalog_page.html

```diff
@@ -5,7 +5,7 @@
 {% block title %}Catalog | {{ site_name }}{% endblock %}
 
 {% block headercontent %}
-  <div id="header"></div>
+  <div id="header" style="height: 98px"></div>
```

Comment:
I'm pretty sure we can just add this to the `#header` selector in `static/scss/detail/header.scss` instead of inlining it in all of these files

Reply:
Looks like a great suggestion, I'll change/test it in CSS and will update the PR shortly.